### PR TITLE
Add support for new endpoint in ruby client [ch26459]

### PR DESCRIPTION
**Pull request by macharmi**

### CODE REVIEW

File: lib/chartmogul/batch/import.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      def serialize_customers
+        customers.map(&:serialize_for_write)
+      end
+
+      def serialize_cancellations
+        cancellations.map(&:serialize_for_write)
+      end
+
+      def serialize_transactions
+        cancellations.map(&:serialize_for_write)
```
Comment (pkopac) [inline lib/chartmogul/batch/import.rb:35]:
```suggestion
        transactions.map(&:serialize_for_write)
```

File: lib/chartmogul/batch/import.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      end
+
+      def serialize_line_items
+        cancellations.map(&:serialize_for_write)
```
Comment (pkopac) [inline lib/chartmogul/batch/import.rb:39]:
```suggestion
        line_items.map(&:serialize_for_write)
```

**Review by pkopac — CHANGES_REQUESTED** [review]
You might want to add a test on the serialized JSON (careful about keys order, use [include](https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/include-matcher)).

File: lib/chartmogul/csv/plan.rb [hunk 3]
```diff
@@ -2,21 +2,19 @@
...
-    PLAN_FIELDS = %i[name interval_count interval_unit external_id].freeze
-    PLAN_HEADERS = %w[Name Interval\ count Interval\ unit Plan\ ID].freeze
+    class Plan < Base
+      # from https://chartmogul-samples.s3-eu-west-1.amazonaws.com/public/02_Plans.csv
+      PLAN_HEADERS = %w[Name Interval\ count Interval\ unit Plan\ ID].freeze
 
-    Plan = Struct.new(*ChartMogul::CSV::PLAN_FIELDS) do
-      include BaseStruct
-      extend CSVHeader
+      # https://github.com/chartmogul/platform/blob/main/engines/data_ingestion/app/services/data_ingestion/csv_mapping/plan.rb
```
Comment (blakemcintyre) [inline lib/chartmogul/csv/plan.rb:9]: We shouldn't have references to internal code in a public repository.

**Review by blakemcintyre — COMMENTED** [review]
What's the plan for this PR as `'/v2/import/:data_source_uuid` currently does nothing other than return `OK`?

**Comment by macharmi** [comment]
@blakemcintyre this is on hold for the moment. @viktorchukhantsev is calling directly DataIngestion classes from integrations instead of making an API rest call.
Please let me know if you need it and I can finish it.

**Pull request closed without merging**